### PR TITLE
[tools/onert_train] Print trainable ops

### PR DESCRIPTION
**Pull request by zetwhite**
This PR make onert_train prints trainable_ops as log.

ONE-DCO-1.0-Signed-off-by: SeungHui Youn <sseung.youn@samsung.com>

### CODE REVIEW

File: tests/tools/onert_train/src/nnfw_util.cc [hunk 1]
```diff
@@ -109,6 +109,16 @@ std::ostream &operator<<(std::ostream &os, const nnfw_train_info &info)
...
+  os << "- trainalbe_ops   = [";
+  auto const size = info.trainble_ops_size;
+  if (size > 0)
+  {
+    for (int i = 0; i < size - 1; ++i)
+      os << info.trainble_ops_idx[i] << ", ";
+    os << info.trainble_ops_idx[size - 1];
+  }
+  os << "]\n";
+
```
Comment (zetwhite) [inline tests/tools/onert_train/src/nnfw_util.cc:121]: printing example 

```bash
➜ ~/Workspace/ONE_second/Product/build/bin/onert_train \
--path mnist.circle \
--load_input:raw mnist_x_train.1000.bin \
--load_expected:raw mnist_y_train.1000.bin \
--trainable_ops_idx 0-3
Model Expected Filename mnist_y_train.1000.bin
Model Input Filename mnist_x_train.1000.bin
Model Filename mnist.circle
== training parameter ==
- learning_rate   = 0.01
- batch_size      = 32
- loss_info       = {loss = categorical crossentropy, reduction = sum over batch size}
- optimizer       = adam
- trainalbe_ops   = [0, 1, 2, 3]
========================
...
```

Comment (zetwhite) [reply tests/tools/onert_train/src/nnfw_util.cc:121]:
```bash
➜ ~/Workspace/ONE_second/Product/build/bin/onert_train \
--path mnist.circle \
--load_input:raw mnist_x_train.1000.bin \
--load_expected:raw mnist_y_train.1000.bin  

Model Expected Filename mnist_y_train.1000.bin
Model Input Filename mnist_x_train.1000.bin
Model Filename mnist.circle
== training parameter ==
- learning_rate   = 0.01
- batch_size      = 32
- loss_info       = {loss = categorical crossentropy, reduction = sum over batch size}
- optimizer       = adam
- trainalbe_ops   = []
========================
```

**Review by jyoungyun — APPROVED** [review]
LGTM

(Please print the `trainable_ops_idx` in the `circle_plus_gen` tool. 😄)

**Review by ragmani — APPROVED** [review]
LGTM

**Pull request merged**